PartymodeAutostart.py

```diff
@@ -57,6 +62,9 @@ def runPartyMode(self):
 
         self.activateVisualisation()
 
+        if (self.volumeBarPartyMode):
+            xbmc.executebuiltin("XBMC.SetVolume(%d,true)" % self.volumePartyMode)
+
```

Comment:
Why do you set the volume two times? and the second time you show the volume dialog?